core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -106,6 +155,13 @@ export class WorkflowActionService {
     this.handleJointLinkAdd();
     this.handleJointOperatorDrag();
     this.handleHighlightedElementPositionChange();
+    this.handleRemoteChange();
+  }
+
+  public undoredo(): void {}
```

Comment:
is this used? please leave some comments

Reply:
Yes, done.